frontend/packages/console-shared/src/utils/icon-utils.ts

```diff
@@ -0,0 +1,7 @@
+import * as _ from 'lodash';
+import * as operatorLogo from '../images/operator.svg';
+
+export const getImageForCSVIcon = (csv) => {
```

Comment:
Type csv? It's a `K8sResourceKind` in your tests, is there a definition shared we can use? Something like `ClusterServiceVersionKind`? Can we import from the `operator-lifecycle-manager`? Is this the right type?

Reply:
Yes, that is the correct type. Updated it..

Reply:
So after talking to Christian, he suggests we do not make this dependency. It pollutes what the `shared` package is trying to do. So in order to keep this, he suggested putting it into `dev-console`. Does that seem viable?